component/main.jsonnet

```diff
@@ -59,9 +59,9 @@ local admin_secret = kube.Secret(params.admin.secretname) {
 local connection_secrets = {
   builtin: {
     // PostgreSQL admin password
-    'postgresql-postgres-password': params.database.password,
+    'postgres-password': params.database.password,
     // this secret is shared between Keycloak and PostgreSQL
-    'postgresql-password': params.database.password,
+    password: params.database.password,
```

Comment:
do I understand that right, `password` and `postgres-password` are the same (`params.database.password`)?

Reply:
Yes. One is used for Keylocak and one for Postgres.